configs/inspection/Idealista.xml

```diff
@@ -77,7 +74,7 @@
   <inspection_tool class="ConstantNamingConvention" enabled="true" level="ERROR" enabled_by_default="true">
     <option name="onlyCheckImmutables" value="false" />
     <option name="m_regex" value="[A-Z][A-Z_\d]*" />
-    <option name="m_minLength" value="3" />
+    <option name="m_minLength" value="5" />
```

Comment:
Is there some reason to go from 3 to 5 here?
You mean that we should avoid shorter names? TAG object?